app/views/NewServerView/ServerInput/index.js

```diff
@@ -42,8 +43,8 @@ const ServerInput = ({
 	return (
 		<View style={styles.container}>
 			<TextInput
-				label='Enter workspace URL'
-				placeholder='Ex. your-company.rocket.chat'
+				label={I18n.t('Enter_workspace_URL', { defaultValue: 'Enter workspace URL' })}
```

Comment:
No need for `defaultValue`.
The app always falls back to `en`.

Reply:
Removed the defaultValue in this commit.